models/eprop_synapse.h

```diff
@@ -319,9 +319,12 @@ class eprop_synapse : public Connection< targetidentifierT >
   //! Low-pass filtered spiking variable.
   double z_bar_ = 0.0;
 
-  //! Low-pass filtered eligibility trace.
+  //! Low-pass filtered eligibility trace with kappa as decay factor.
```

Comment:
```suggestion
  //! Low-pass filtered eligibility trace.
```
`kappa` and `kappa_reg` are not known in the synapse. This information might be confusing here.